src/program/top/top.lua

```diff
@@ -32,7 +32,9 @@ function run (args)
 end
 
 function select_snabb_instance (pid)
-   local instances = shm.children("/")
+   local instances = {}
+   -- For named programs there is a "by-name" directory which isn't a process, remove it.
```

Comment:
Would you mind doing a little refactor while you're at it -- please add something like
```lua
local function compute_snabb_instances()
   --- Compute the set of Snabb processes, excluding this one.
   local pids = {}
   local my_pid = S.getpid()
   for _,name in ipairs(shm.children('/')) do
      local pid = tonumber(name) -- Could fail for the "by-name" directory.
      if pid and pid ~= my_pid then table.insert(pids, pid) end
   end
   return instances
end
```

and refactor the select_snabb_instance function to be simpler.